src/test/scala/edu/ie3/mobsim/utils/UtilsSpec.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.mobsim.utils
+
+import edu.ie3.test.common.UnitSpec
+
+class UtilsSpec extends UnitSpec {
+
+  "Utils should round to quarter hour successfully" in {
+
+    val cases = Table(
+      ("time", "roundedTime"),
+      (4, 15),
```

Comment:
Why should this be 15 and not zero?

Reply:
If we want to round up to the next quarter hour we might take 
```
  def roundToQuarterHourInMinutes(minutes: Int): Int = {
    val intervals15 = math.ceil(minutes.toDouble / 15).toInt
    intervals15 * 15
```

Reply:
We might add also some test for hitting exactly the quarter hour (e.g. 30 should be 30 or 90 should be 90).

Reply:
Since we use it for calculating the driving time and we have already established, there is trip that takes place driving time can't be 0

Reply:
Also only want to force rounding up in case the time would be zero otherwise. In other cases we would screw with the time distributions when we always round up.